uint16 id
uint16 address
uint16 data_type
float32  value